config/katello.migrations/161130090556-server_ssl_crl_blank.rb

```diff
@@ -0,0 +1,3 @@
+# server_ssl_crl is expected to be an absolute path or a blank string, not
+# false. Katelllo isn't using a CRL, so this needs to be a blank string.
+answers['foreman']['server_ssl_crl'] = ""
```

Comment:
Yea, you're right, I'll check first if it has a value other than false: https://github.com/Katello/katello-installer/pull/444